src/interface/RHInterface.py

```diff
@@ -204,39 +219,64 @@ def update(self):
                     lap_id = data[0]
                     lap_time_ms = 0
 
-                    rssi_val = unpack_16(data[5:])
+                    if node.api_level >= 18:
+                        offset_rssi = 3
```

Comment:
Double check these offsets; I think they're not quite right.

Reply:
Wow, I was quite abit off.